test/integration/config/index.tap.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const Config = require('../../../lib/config')
+const tap = require('tap')
+
+tap.test('Configuration Library', function (harness) {
+  harness.autoend()
```

Comment:
As a matter of style, we just call this parameter `t`.

We also seem to have a strong preference over arrow functions instead of the `function` keyword.